lib/Runtime/Library/JSONStringifier.cpp

```diff
@@ -409,10 +402,49 @@ JSONStringifier::ReadObjectElement(
             // If gap is specified, a space is appended
             UInt32Math::Inc(this->totalStringLength);
         }
-        jsonObject->Push(prop);
     }
 }
 
+void
+JSONStringifier::ReadObjectElement(
+    _In_ JavascriptString* propertyName,
+    _In_ uint32 numericIndex,
+    _In_ RecyclableObject* obj,
+    _In_ JSONObject* jsonObject,
+    _In_ JSONObjectStack* objectStack)
+{
+    JSONObjectProperty prop;
+    prop.propertyName = propertyName;
+
+    Var value = JavascriptOperators::GetItem(obj, numericIndex, this->scriptContext);
```

Comment:
>:: [](start = 35, length = 2)

Fantastic wins.

So if I understand - ```obj``` is typed array, so this will call to the DirectGetItem of typed array which will have unnecessary checks being detached and others . Could that be avoided (or it is already avoided)?